.bingo/mockgen.mod

```diff
@@ -1,5 +1,5 @@
 module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
-go 1.21
+go 1.21.13
```

Comment:
Goutham is trying to upgrade us to go 1.22. I see they've added support for 22 recently. Any reason to merget this at 1.21 and then update to 1.22?

Reply:
Discussed in Slack. We decided to merge this PR as-is since it's blocking stuff, and I'll branch off of Goutham's 1.22 working branch to prepare another 1.22 PR related to this stuff as needed.